automatic/chromium/tools/chocolateyInstall.ps1

```diff
@@ -14,7 +13,7 @@ $validExitCodes = @(0)
   if (Test-Path $Chromium) {
     $silentArgs = ''
   } else {
-    $silentArgs = '--system-level --do-not-launch-chrome'
+    $silentArgs = '--system-level'
   }
   
-    Install-ChocolateyPackage $packageName $fileType $silentArgs $url $validExitCodes $checksum $checksumType
+    Install-ChocolateyPackage $packageName $fileType $silentArgs $url $checksum $checksumType

```

Comment:
@ferventcoder so would changing the order to be
}
   
-    Install-ChocolateyPackage $packageName $fileType $silentArgs $checksum $checksumType $url

Make chocolatey pass the variables correctly, or should I just pull the whole check for bitness and let chocolatey determine it at run time?